REQUESTLOG.MARCJOSEPH.md

```diff
@@ -0,0 +1,15 @@
+# How to pull request
+1. (Date & Time of Request) i.e "10/23/2017 5:45 pm" (Passenger & Destination in Question) "Regina to Margerita's Pizzeria" add date and and estimated time of pickup "at/for 10:45 pm 10/24/2017" add the request bracket and put "| requested []" the parent then approves or denies the request if denial is given a reason has to be given and can be up for a dispute a reason can look like this "reason: too late over curfew & short notice."
```

Comment:
Kind of confusing -- lets go with this

person[s]: <person name> 
event: <event title>
location [1/2/3] / title[pick up/ drop off/ accompany]: <location address> <location time range>